backend/src/main/java/com/funeat/recipe/presentation/RecipeApiController.java

```diff
@@ -88,4 +92,22 @@ public ResponseEntity<SearchRecipeResultsResponse> getSearchResults(@RequestPara
 
         return ResponseEntity.ok(response);
     }
+
+    @PostMapping("/api/recipes/{recipeId}/comments")
+    public ResponseEntity<Void> writeComment(@AuthenticationPrincipal final LoginInfo loginInfo,
+                                             @PathVariable final Long recipeId,
+                                             @RequestBody @Valid final RecipeCommentCreateRequest request) {
+        final Long savedCommentId = recipeService.writeCommentOfRecipe(loginInfo.getId(), recipeId, request);
+
+        return ResponseEntity.created(URI.create("/api/recipes/" + recipeId + "/" + savedCommentId)).build();
+    }
+
+    @GetMapping("/api/recipes/{recipeId}/comments")
+    public ResponseEntity<RecipeCommentsResponse> getCommentsOfRecipe(
+            @AuthenticationPrincipal final LoginInfo loginInfo, @PathVariable final Long recipeId,
+            @ModelAttribute final RecipeCommentCondition condition) {
```

Comment:
```suggestion
            @AuthenticationPrincipal final LoginInfo loginInfo,
            @PathVariable final Long recipeId,
            @ModelAttribute final RecipeCommentCondition condition) {
```
얘도 정렬 이렇게 하는거 어떨까요!!

Reply:
이부분은 저희 컨벤션대로 wooteco.xml 로 설정한 거라서 그냥 가는게 좋을 것 같아요!